src/common/Constants.h

```diff
@@ -30,6 +30,7 @@ const unsigned int BLOCK_HASH_SIZE = 32;
 const unsigned int BLOCK_SIG_SIZE = 64;
 
 // Numeric types sizes
+const unsigned int UNSIGNEDCHAR_SIZE = 1;
```

Comment:
Why need to define this instead of using sizeof(unsigned char)?

Reply:
@jiayaoqijia pointed out that sizeof(unsigned char) or sizeof(unsigned int) may different on different OS. Hence, it is good that we fixed the length.

Reply:
https://www.tutorialspoint.com/cprogramming/c_data_types.htm

This page shows only sizeof(int) and sizeof(unsigned int) will differ on different OS.

Reply:
Good catch, although I would refer to the c++ [standard](http://www.open-std.org/jtc1/sc22/wg21/docs/papers/2014/n4296.pdf) **5.3.3 Sizeof**

Reply:
We probably want to fix this